spec/descriptions/tagSyntheticResults.md

```diff
@@ -1,6 +1,12 @@
 The endpoints of this group retrieve the results for defined Synthetic tests.
```

Comment:
There is a line at the very top of this file that still mentions Tech Preview.  That whole sentence should be removed.